albumentations/augmentations/functional.py

```diff
@@ -876,16 +876,42 @@ def gauss_noise(image, gauss):
     return image + gauss
 
 
-@clipped
-def brightness_contrast_adjust(img, alpha=1, beta=0):
+def _brightness_contrast_adjust_non_unit(img, alpha=1, beta=0):
     img = img.astype('float32')
+
     if alpha != 1:
         img *= alpha
     if beta != 0:
         img += beta * np.mean(img)
     return img
 
 
+@preserve_shape
```

Comment:
why do we need to preserve shape here?

Reply:
`img = cv2.LUT(img, lut)`

changes the shape from `(1, N, N)` to `(N, N)`